applications/nrf_desktop/README.rst

```diff
@@ -965,6 +965,22 @@ Apart from the device role, the application-specific Kconfigs specify a set of s
 By default, the nRF Desktop devices use predefined format of HID reports.
 The common HID report map is defined in the :file:`configuration/common/hid_report_desc.c` file.
 
+Debug configuration
+-------------------
+
+The nRF Desktop application introduces application-specific configuration options related to Debug configuration.
+These options are defined in :file:`Kconfig.debug` file.
+
+The :kconfig:option:`CONFIG_DESKTOP_LOG` Kconfig option enables support for logging in the nRF Desktop application.
+This option overlays Kconfig option defaults from the Logging subsystem to align them with the nRF Desktop requirements.
+The nRF Desktop configuration uses Segger J-Link RTT as the Logging subsystem backend.
+
+The :kconfig:option:`CONFIG_DESKTOP_SHELL` Kconfig option enables support for CLI in the nRF Desktop application.
+This option overlays Kconfig option defaults from the Shell subsystem to align them with the nRF Desktop requirements.
+The nRF Desktop configuration uses Segger J-Link RTT as the Shell subsystem backend.
```

Comment:
`If both shell and logging are enabled, logger uses shell as the logging backend.`